src/extension/browserAction/about.html

```diff
@@ -0,0 +1,32 @@
+<html>
+
+<head>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <link rel="stylesheet" href="normalize.css">  
+  <link rel="stylesheet" href="styles2FAN.css">
+  <link href="https://fonts.googleapis.com/css?family=Rubik:300,400,700,900" rel="stylesheet">
+  <script
+  src="./thirdParty/jquery-3.3.1.min.js"
+  integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8="
+  crossorigin="anonymous"></script>
+  <script src="./supports2FA.js"></script>
+</head>
+
+<body>
+  <a href="menu.html" class="icon primary-nav">
+    <svg width="24" height="24" viewBox="0 0 24 24" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" class="icon">
+      <path d="M 0.404223 9.95959L 8.0766 17.5978C 8.61488 18.1341 9.48901 18.1341 10.0269 17.5978C 10.5659 17.0613 10.5659 16.1117 10.0269 15.5755L 4.88179 10.3726L 18.6207 10.3726C 19.3824 10.3726 20 9.75773 20 8.9994C 20 8.24107 19.3824 7.62623 18.6207 7.62623L 4.53696 7.62623L 9.98384 2.28358C 10.5228 1.74736 10.5228 0.917963 9.98384 0.382084C 9.71487 0.113973 9.36142 -5.23822e-08 9.00867 -5.23822e-08C 8.65591 -5.23822e-08 8.30246 0.144183 8.0335 0.412294L 0.404223 8.01278C -0.13474 8.54866 -0.13474 9.42337 0.404223 9.95959Z" transform="translate(2 3)"/>
+    </svg>
+  </a>
+  <section class="extension-body-text">
+    <p>Created by <a href="https://www.allthingsauth.com/">Conor</a> and <a href="https://ray-gonzales.com">Ray</a>, 2FAN (Two Factor Authentication Notifier) strives to help all netizens be more secure on the web.</p> 
+
+    <p>Thanks to the folks at <a href="https://twofactorauth.org/">twofactorauth.org</a> for opensourcing their project, which allowed us to build this tool.</p>
+
+    <p>Also, thanks to Blair Adams, Perametade Games, anbileru adaleru for allowing us to use their icons through <a href="https://thenounproject.com">thenounproject.com</a> as well as the <a href="https://material.io/icons/">Google Design team</a> for open sourcing their icons.</p>
+    
+    <p>If you are interested in jumping into the code for 2FAN, we’ve open sourced everything on our <a href="https://github.com/conorgil/2fa-notifier">Github repo here.</a></p>
```

Comment:
Weird quote character renders all funny looking. Can you change to regular quote like you did on the feedback page?